src/axolotl/utils/models.py

```diff
@@ -127,6 +127,14 @@ def load_model(
         # TODO: Check if this would overwrite previous additional_special_tokens
         tokenizer.add_special_tokens({"additional_special_tokens": [MEM_TOKEN]})
 
+    if cfg.is_llama_derived_model and cfg.xpos_rope:
```

Comment:
Perhaps it should be `elif` from above. We should maybe not allow multiple patches?

Reply:
it works with xformers_attention,it's specific to the rotary embedding layer which most of the above doesn't touch

Reply:
Oh ok!

Reply:
Could you also add this to Readme please?